src/msbuild/src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1883,16 +1883,30 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <!-- For each reference, get closest match -->
 
-    <!-- Pass the CurrentProjectTargetPlatform parameter to the task only if GetReferenceNearestTargetFrameworkTaskSupportsTargetPlatformParameter is true.  This means
+    <!-- Pass the CurrentProjectTargetFrameworkProperty parameter to the task only if GetReferenceNearestTargetFrameworkTaskSupportsTargetFrameworkPropertyParameter is true.  This means
          that we are using a version of NuGet which supports that parameter on this task. -->
 
     <GetReferenceNearestTargetFrameworkTask AnnotatedProjectReferences="@(_ProjectReferenceTargetFrameworkPossibilities)"
                                             CurrentProjectTargetFramework="$(ReferringTargetFrameworkForProjectReferences)"
                                             CurrentProjectTargetPlatform="$(TargetPlatformMoniker)"
+                                            CurrentProjectTargetFrameworkProperty="$(TargetFramework)"
```

Comment:
Would you mind describing the difference between this parameter and the `CurrentProjectTargetFramework` one?